pkg/cluster/internal/kubeadm/config.go

```diff
@@ -439,6 +451,17 @@ func Config(data ConfigData) (config string, err error) {
 	// derive any automatic fields if not supplied
 	data.Derive()
 
+	// Kubeadm has its own feature-gate for dual stack
+	// we need to enable it for Kubernetes version 1.20 only
+	// dual-stack is only supported in 1.20+
+	// TODO: remove this when 1.20 is EOL or we no longer support
+	// dual-stack for 1.20 in KIND
+	if ver.LessThan(version.MustParseSemantic("v1.21.0")) &&
```

Comment:
we should really use the version for the PR that enabled it by default so it works with arbitrary commits 🙃 

if you checkout kubernetes master then the commit you can grab the version.

Reply:
can be a follow-up

Reply:
this was the kubeadm one https://github.com/kubernetes/kubernetes/pull/99294

and this the kubernetes one https://github.com/kubernetes/kubernetes/pull/98969

Reply:
the must parse semantic doesn't accept versions with letters IIRC